tensorflow/core/BUILD

```diff
@@ -1150,7 +1150,7 @@ cc_library(
     ],
     copts = tf_copts(),
     defines = tf_additional_lib_defines(),
-    linkopts = ["-ldl"],
```

Comment:
Would it be possible to have a config_setting() rule and a select() function to selectively apply this change to only freebsd? This might work, but I'm not sure.

```python
config_setting(
    name = "freebsd",
    values = {"cpu": "freebsd"},
)
```

See also: https://github.com/tensorflow/tensorflow/blob/master/third_party/jpeg/jpeg.BUILD#L463